x-pack/solutions/observability/plugins/observability_ai_assistant_app/server/functions/alerts.ts

```diff
@@ -195,7 +195,9 @@ export function registerAlertsFunction({
               filter: [
```

Comment:
@sorenlouv any thoughts WRT testing this? My initial thought is to breakout the anonymous function call that's registered into a separate function, then just unit test that to ensure the right field is queried.

Reply:
We have a test for this in https://github.com/elastic/kibana/blob/main/x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/complete/functions/alerts.spec.ts

We currently don't generate any alerts to test against - we should. 

cc @viduni94

Reply:
So to answer your question: I'd prefer extending the API tests rather than adding a unit test